modules/users/client/views/admin/view-user.client.view.html

```diff
@@ -8,7 +8,7 @@ <h1 ng-bind="vm.user.username"></h1>
         <a class="btn btn-primary" ui-sref="admin.user-edit({userId: vm.user._id})">
           <i class="glyphicon glyphicon-edit"></i>
         </a>
-        <a class="btn btn-primary" ng-click="vm.remove()" ng-if="vm.user._id !== vm.authentication.user._id">
+        <a class="btn btn-primary" ng-click="vm.remove()" ng-if="vm.user.username !== vm.authentication.user.username">
```

Comment:
@mleanos did you mean client controller or server controller? reason I ask is the existing 'isCurrentUserOwner' is done on the server controller. I believe you mean client controller as you referred vm and I am ok with that approach. Just wanted to make sure you are ok if it is not consistent across